src/Cake/Scripting/Roslyn/Installation/RoslynInstallerInstructions.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using Cake.Core.IO;
+using NuGet;
+
+namespace Cake.Scripting.Roslyn.Installation
+{
+    /// <summary>
+    /// Represents instructions for installing a NuGet package
+    /// and copy specific files to the bin directory.
+    /// </summary>
+    public sealed class RoslynInstallerInstructions
+    {
+        private readonly List<FilePath> _paths;
+        private readonly List<Tuple<string, SemanticVersion>> _packages;
+        private readonly List<PackageSource> _packageSources;
+
+        /// <summary>
+        /// Gets the paths to the files that should be copied.
+        /// </summary>
+        /// <value>The paths.</value>
+        public IReadOnlyList<FilePath> Paths
+        {
+            get { return _paths; }
+        }
+
+        /// <summary>
+        /// Gets the packages to be installed.
+        /// </summary>
+        /// <value>The packages.</value>
+        public IReadOnlyList<Tuple<string, SemanticVersion>> Packages
```

Comment:
Actually, A `Dictionary<string, SemanticVersion>` would be even better. Think I need some sleep... :smile: